diff -urN aa/libgnome-java-2.5.4/configure.in libgnome-java-2.5.4/configure.in
--- aa/libgnome-java-2.5.4/configure.in	2004-02-04 01:55:17.000000000 +0100
+++ libgnome-java-2.5.4/configure.in	2004-02-08 21:21:47.918767104 +0100
@@ -35,6 +35,9 @@
 
 AC_INIT(src/jni/org_gnu_gnome_Gnome.c)
 
+AC_SUBST(version)
+AC_SUBST(apiversion)
+AC_SUBST(gtkapiversion)
 
 dnl Find already installed gnome modules
 
