diff -urN aa/libgnome-java-2.5.5/aclocal.m4 libgnome-java-2.5.5/aclocal.m4
--- aa/libgnome-java-2.5.5/aclocal.m4	2004-02-17 02:38:56.000000000 +0100
+++ libgnome-java-2.5.5/aclocal.m4	2004-02-21 22:15:36.787277640 +0100
@@ -688,7 +688,9 @@
         GCJ_HOME=`which gcj | sed 's/\(.*\).bin.*gcj/\1/'`
      fi
 
-     GCJ_JAR=`locate libgcj | grep libgcj.*\.jar`
+     if test x$GCJ_JAR = x ; then
+       GCJ_JAR=`locate libgcj | grep libgcj.*\.jar`
+     fi
      if test x$GCJ_JAR != x ; then
         GCJ_CLASSPATH="${GCJ_JAR}:${CLASSPATH}"
      else
